docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+| ACME “tls-alpn-01” method for IP Addresses (defined in 3.2.2.5.7) | This method involves establishing a TLS connection.<br><br>
+“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+
+
+Table: Defining Corroborating CAA Evidence
+
+| CAA Method | Description of Corroborating Evidence |
+|---|---|
+| Any method | “Corroborate” means the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table MUST have CAA lookup results that permit issuance for the domain in question. |
+
+Computing systems performing Multi-Perspective Domain Validation:
+
+- MUST:
+    - Forward all Internet traffic via a network or set of networks that filter all RPKI-invalid BGP routes as defined by RFC 6811 
+
+- SHOULD:
+    - Facility & Service Provider Requirements
+      - Be hosted from an ISO/IEC 27001 certified facility or equivalent. 
+      - Rely on services covered in one of the following reports: System and Organization Controls 3 (SOC 3), IASE 3000, ENISA 715, FedRAMP Moderate, C5:2020, CSA STAR CCM, or equivalent.
+    - Network Hardening
+      - Configure each network boundary control (firewall, switch, router, gateway, or other network control device or system) with rules that support only the services, protocols, ports, and communications identified as necessary to its operations.
+    - System Hardening
+      - Disable all accounts, applications, services, protocols, and ports that are not used.
+      - Implement multi-factor authentication for all user accounts.
+    - Vulnerability Detection and Patch Management
+      - Implement intrusion detection and prevention controls to protect against common network and system threats.
+      - Document and follow a vulnerability correction process that addresses the identification, review, response, and remediation of vulnerabilities.
+      - Undergo or perform a Vulnerability Scan at least every 3 months.
+      - Undergo a Penetration Test on at least an annual basis
+      - Apply recommended security patches within six (6) months of the security patch’s availability, unless the CA documents that the security patch would introduce additional vulnerabilities or instabilities that outweigh the benefits of applying the security patch.
+
+Beyond the above considerations, computing systems performing Multi-Perspective Domain Validation are considered outside of the audit scope described in Section 8 of this Policy.
+
```

Comment:
This sentence implies that the "above considerations" MUST be within the audit scope described in Section 8, i.e. require WebTrust for CAs or ETSI audits from Network Perspectives. If these Network Perspectives are hosted by a Delegated Third Party cloud provider, I don't think the intent is to enforce these DTPs to go through a WebTrust or ETSI audit.

The CA must be able to obtain certifications and/or some form of assurance that these "above considerations" are followed by a third party hosting each Network Perspective, to satisfy these requirements.

How about:

> If any of the above considerations are performed by a Delegated Third Party, the CA MAY obtain certifications and/or some form of assurances from the Delegated Third Party that one or more of the above considerations are followed. 

> Delegated Third Parties need not be within the audit scope described in Section 8 of these Requirements, to satisfy the above considerations.

Reply:
Any issues with: "_If any of the above considerations are performed by a Delegated Third Party, the CA MAY obtain reasonable evidence from the Delegated Third Party to ascertain assurance that one or more of the above considerations are followed. Delegated Third Parties are not required to be within the audit scope described in Section 8 of these Requirements to satisfy the above considerations._"

"Reasonable evidence" seems simpler (to me) than trying to describe what that evidence might/should look like. We also use the concept of "reasononable" elsewhere in the BRs.

Pushed in https://github.com/ryancdickson/staging/commit/e9d7c8996f3cbf9f406f82b8374ef4dc235d0b76